vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridPreloadIT.java

```diff
@@ -128,15 +128,13 @@ public void firstExpanded_scrollByViewportEstimate_shouldPreLoadDataForExpandedC
     }
 
     @Test
-    public void firstExpanded_reExpand_shouldPreLoadDataForExpandedChildren() {
```

Comment:
Seems that there's a difference if the expand item command originates from the server vs the client. 0 additional requests is better than 2 so repurposed these re-expand tests. There are other tests that validate that the content is correct after re-expanding.